plugins/vsphere/pkg/command/login.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2020 VMware, Inc.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package command
+
+import (
+	"fmt"
+	"io/ioutil"
+	"syscall"
+
+	"golang.org/x/crypto/ssh/terminal"
+
+	"github.com/spf13/cobra"
+	"github.com/vmware-tanzu/sources-for-knative/plugins/vsphere/pkg"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type LoginOptions struct {
+	Namespace     string
+	Username      string
+	Password      string
+	SecretName    string
+	PasswordStdIn bool
+}
+
+func NewLoginCommand(clients *pkg.Client) *cobra.Command {
+	options := &LoginOptions{}
+
+	result := &cobra.Command{
+		Use:   "login",
+		Short: "Create the required vSphere credentials",
+		Long: `Create the required vSphere credentials
+
+Examples:
+# Log in the default namespace
+kn vsphere login --username jane-doe --password s3cr3t --secret-name vsphere-credentials
+# Log in the specified namespace
+kn vsphere login --namespace ns --username john-doe --password s3cr3t --secret-name vsphere-credentials
+# Log in the specified namespace with the password retrieved via standard input
+kn vsphere login --namespace ns --username john-doe --password-stdin --secret-name vsphere-credentials
```

Comment:
There is an "Example" field for these, right?

Reply:
Oops, yes! 😅